tools/dm-query-status.md

```diff
@@ -0,0 +1,234 @@
+                        "totalBytes": "452",    # The total number of bytes that need to be imported.
+                        "progress": "25.44 %"   # The progress of the full import.
+                    }
+                }
+            ],
+            "relayStatus": {
+                "masterBinlog": "(bin.000001, 28507)",
+                "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
+                "relaySubDir": "c0149e17-dff1-11e8-b6a8-0242ac110004.000001",
+                "relayBinlog": "(bin.000001, 28507)",
+                "relayBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-96",
+                "relayCatchUpMaster": true,
+                "stage": "Running",
+                "result": null
+            }
+        }
+                {
+            "result": true,
+            "worker": "172.17.0.6:10081",
+            "msg": "",
+            "subTaskStatus": [
+                {
+                    "name": "test",
+                    "stage": "Paused",
+                    "unit": "Load",
+                    "result": {                 # The error example.
+                        "isCanceled": false,
+                        "errors": [
+                            {
+                                "Type": "ExecSQL",
+                                "msg": "Error 1062: Duplicate entry '1155173304420532225' for key 'PRIMARY'\n/home/jenkins/workspace/build_dm/go/src/github.com/pingcap/tidb-enterprise-tools/loader/db.go:160: \n/home/jenkins/workspace/build_dm/go/src/github.com/pingcap/tidb-enterprise-tools/loader/db.go:105: \n/home/jenkins/workspace/build_dm/go/src/github.com/pingcap/tidb-enterprise-tools/loader/loader.go:138: file test.t1.sql"
+                            }
+                        ],
+                        "detail": null
+                    },
+                    "unresolvedDDLLockID": "",
+                    "load": {
+                        "finishedBytes": "0",
+                        "totalBytes": "156",
+                        "progress": "0.00 %"
+                    }
+                }
+            ],
+            "relayStatus": {
+                "masterBinlog": "(bin.000001, 1691)",
+                "masterBinlogGtid": "97b5142f-e19c-11e8-808c-0242ac110005:1-9",
+                "relaySubDir": "97b5142f-e19c-11e8-808c-0242ac110005.000001",
+                "relayBinlog": "(bin.000001, 1691)",
+                "relayBinlogGtid": "97b5142f-e19c-11e8-808c-0242ac110005:1-9",
+                "relayCatchUpMaster": true,
+                "stage": "Running",
+                "result": null
+            }
+        }
+    ]
+}
+
+```
+
+For the status description and status switch relationship of "stage" of "subTaskStatus" of "workers", see [Subtask status](#subtask-status).
+For operation details of "unresolvedDDLLockID" of "subTaskStatus" of "workers", see [Troubleshooting Sharding DDL Locks](../tools/troubleshooting-sharding-ddl-locks/).
```

Comment:
Where do LINE 184-185 come from, I can't find them in CN docs.
https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/task-handling/query-status.md

Reply:
@amyangfei They are in the above code block. Relative URLs are not user-friendly in the code block, so I move them out of this block and place them here.